Web/web-example-produce.html

```diff
@@ -28,17 +44,17 @@
                 random: Math.random()
             }
 
+            if (!client) {
```

Comment:
Checking for client in case user didn't replace stream id or private key to prevent an error from being thrown